docs/data/material/components/menus/menus.md

```diff
@@ -17,13 +17,19 @@ A menu displays a list of choices on a temporary surface. It appears when the us
 
 {{"component": "@mui/docs/ComponentLinkHeader"}}
 
+## Introduction
+
+Menus are implemented using a collection of related components:
+
+- Menu: The container/surface of the menu.
+- Menu Item: List item option for the menu.
```

Comment:
Reading "List item" makes me think Menu Items always goes inside Menu List.

Reply:
Good point, I'll reword this to avoid that implication. How about this:

```suggestion
- Menu Item: An option for users to select from the menu.
```

Reply:
Nice!